applications/tari_validator_node/src/grpc/services/wallet_client.rs

```diff
@@ -72,11 +72,11 @@ impl WalletClient for GrpcWalletClient {
         contract_id: &FixedHash,
         state_root: &StateRoot,
         checkpoint_number: u64,
-        checkpoint_signatures: &[SignerSignature],
+        checkpoint_signatures: Vec<SignerSignature>,
```

Comment:
Why change this to a vec?
We are not changing this in the function?

Reply:
I am sorry @SWvheerden, this was due to a merge conflict that I missed.